src/nifake/tests/test_session.py

```diff
@@ -1,12 +1,72 @@
-import math
+import ctypes
+# import math
 import mock_helper
 import nifake
-import sys
-import warnings
+# import sys
+from nifake import visatype
+# import warnings
 
-from mock import ANY
+# from mock import ANY
 from mock import patch
 
+
+# Matching classes for setting mock expectations:
```

Comment:
Can/Should these matcher helpers be moved to somewhere else to single source them? Add them to mock_helper and rename it to unit_test_helper?

Reply:
Moved to matchers.py